src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallDao {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    Optional<CinemaHall> get(Long id);
+
+    List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method to `void` or returning the updated `CinemaHall` entity if applicable. Returning a `Boolean` might not be necessary if exceptions are used to indicate failure.